__tests__/app.test.js

```diff
@@ -115,3 +117,43 @@ describe("Error Handling 404 ", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("return 200 status, should get a comment getby its article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toHaveProperty("comment_id");
+
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+
+        expect(article.article_id).toBe(1);
+      });
+  });
+});
+describe("Error Handling 400/404", () => {
+  test("return 404 status, should return an error when the article is not found", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toHaveProperty("message");
+        expect(body.message).toBe("Article not found");
+      });
+  });
+  test("return 404 status, should return an error for not valid article id", () => {
+    return request(app)
+      .get("/api/articles/no/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body).toHaveProperty("message");
+        expect(body.message).toBe("Bad request");
+      });
+  });
+});
```

Comment:
Nice work so far on this! Here's a couple of suggestions to work on to add to this testing suite:
- We need to ensure that the comments are served with the most recent first, so will require some sorting and testing for that.
- We also need to distinguish between an article id not found and article id found but no comments on it. We may need to check if the article id exists first before trying to grab the comments.